static/sass/_snapcraft_p-tabs.scss

```diff
@@ -1,11 +1,21 @@
 @mixin snapcraft-p-tabs {
   @include vf-p-tabs;
 
-  .p-tabs {
+  .p-tabs-bordered {
     @extend .p-tabs; // sass-lint:disable-line placeholder-in-extend
     box-shadow: none;
```

Comment:
I think that Vanilla tabs don't have shadow anymore, so we don't need this anymore.

Reply:
You mean in new designs? Will ask Lyubo when he's around

Reply:
Removed! Didn't understand what you meant, was looking at the `-bordered` addition highlighted in green, not the line that you added the comment to. Silly me